tools/export/iar/iar_definitions.json

```diff
@@ -278,5 +278,8 @@
         "OGChipSelectEditMenu": "TMPM46BF10FG\tToshiba TMPM46BF10FG",
         "GFPUCoreSlave": 21,
         "GBECoreSlave": 21
+    },
+    "M2351KIAAEES": {
+        "OGChipSelectEditMenu": "M2351 series\tNuvoton M2351 series"
```

Comment:
As this exporter works with IAR7, I don't see how this could possibly be useful.

Reply:
@theotherjimmy Need any modification with it? I am not familiar with export.

Reply:
@ccli8 I'm confused by your question. What are you asking? Could you say that another way?

Reply:
>  Need any modification with it?

Yes. @theotherjimmy meant since we are not supporting IAR8.2 in mbed-os addition of M2351KIAAEES will not help (M2351KIAAEES is not supported by IAR7). Please remove this change as it not needed.

Reply:
@theotherjimmy @deepikabhavnani I remove **M2351KIAAEES** from `iar_definitions.json` in https://github.com/ARMmbed/mbed-os/pull/7302/commits/1dbdcb2c8b2461ecd55617271827b9e163df3d27.